javascript-conditionals/javascript-conditionals-quiz-notes.md

```diff
@@ -7,7 +7,7 @@ After completing this exercise, you should be able to discuss or answer the foll
 - Give 6 examples of comparison operators.
 Greater than >, Less than <, Equal to ==, Stricly equal to ===, Not equal to !=, Less than or equal to <=
 - What data type do comparison expressions evaluate to?
-Numbers.
+Numbers, strings, values and variables.
```

Comment:
Still nope! It only evaluates to one data type, not multiple. Btw values and variables aren't a data type.

In the image below, what does it say condition will evaluate to?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else

<img width="785" alt="Screenshot 2023-03-22 at 4 14 03 PM" src="https://user-images.githubusercontent.com/71413368/227059317-b75d4ed1-208a-4bc5-b187-46fc84c0ed7e.png">